notebooks/agent-breakpoints.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p><code>on the tools used by the Agent</code> -&gt; <code>on any of the tools used by the Agent</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>